src/eth/fee_market.yaml

```diff
@@ -6,6 +6,28 @@
     schema:
       title: Gas price
       $ref: '#/components/schemas/uint'
+- name: eth_gasPrices
```

Comment:
I'm a little hesitant to add an aggregate method such as this. I think `eth_blobGasPrice` would fall in line better with the methods `eth_gasPrice` and `eth_maxPriorityFeePerGas`. If the caller needs both, they can submit a JSON-RPC batch call

Reply:
Such straightforward approach is always ok to fallback too, Adding aggregate method is just an attempts to optimize:
- seems like these methods are usually called together to calculate values of transaction fields and use the same data like certain block headers(does not matter a lot because we can cache);
- we may have more gas dimensions in the future and and seems like gasPrices might be a new kind of primitive (vector) to describe gas(but it's not very good to care about distant future things ig);
- less rpc calls(resolved with batch call; but I hope no x3 charge for that).

Special thanks for the feedback about this part, I was not sure whether people might find it more convenient. Maybe blob gas will be queried mostly alone during waiting for cheaper price.